### PR TITLE
New option: "slideChange" for slide change callbacks

### PR DESCRIPTION
**Pull request by archinal**
In this PR I've added an options parameter 'slideChange' to allow users to provide a slide change callback function. This addition has been helpful for me in site I'm developing where we need to trigger custom behaviour elsewhere on the page as the slides are changed.

I've also updated the README and included a new version of the minified JS, which was done using the latest version of [uglify](https://www.npmjs.com/package/uglify), and is actually more compressed than the old minified javascript.

### CODE REVIEW

File: jquery.simplebanner.js [hunk 1]
```diff
@@ -22,7 +22,8 @@
             autoRotate:true,
             maxRotations: null,
             rotateTimeout:5000,
-            animTime:300
+            animTime:300,
+            slideChange: null
```
Comment (EnzoMartin) [inline jquery.simplebanner.js]: I'd prefer to have it called `onChange` to signify it's a function

File: jquery.simplebanner.js [hunk 2]
```diff
@@ -140,6 +141,10 @@
         self.element.find('.bannerList').stop(false,true).animate({
             'marginLeft':-slideIndex * self._bannerWidth
         },self.options.animTime);
+        
+        if(self.options.slideChange !== null){
```
Comment (EnzoMartin) [inline jquery.simplebanner.js]: Should rather check if it's a function, in case it's erroneously set to an object/boolean/etc.

**Review by EnzoMartin — CHANGES_REQUESTED** [review]
Appreciate the PR, just a couple small tweaks!

**Comment by EnzoMartin** [comment]
Also if you can bump the `minor` version in `bower.json`/`package.json`, and optionally add yourself to contributors in the package.json 👍

**Comment by archinal** [comment]
Thanks for the feedback! I've made the changes you suggested. Please let me know if there are any other changes you'd like me to make.

**Review by EnzoMartin — APPROVED** [review]

**Comment by EnzoMartin** [comment]
👍 Awesome!

I've published the new version on NPM, thanks for making the PR!

**Pull request merged**